src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/hooks.js

```diff
@@ -117,19 +118,35 @@ export const resetCardHooks = (updateSettings) => {
   };
 };
 
-export const scoringCardHooks = (scoring, updateSettings) => {
+export const scoringCardHooks = (scoring, updateSettings, defaultValue) => {
+  const [local, setLocal] = module.state.local(scoring.attempts.number);
   const handleMaxAttemptChange = (event) => {
     let unlimitedAttempts = false;
     let attemptNumber = parseInt(event.target.value);
+    const { value } = event.target;
     if (_.isNaN(attemptNumber)) {
```

Comment:
Apparently the attemptNumber used to actually be a number. Now that this has changed because the input type is different, I think we don't want to change the type of the redux state from number to string. So it would be good to change the input from a string into a number before calling `updateSettings`.